### ISSUE TITLE
Generate OTP failed. Beneficiary.validation.wait_to_send_otp

### ISSUE THREAD

**Issue by francisbegbie** [cowinapi/developer.cowin#405]
Whenever I try to sign in using my mobile number, I get the following errors at the bottom:

Beneficiary.validation.wait_to_send_otp
Generate OTP failed

This mobile number has one beneficiary who is partially vaccinated.

I am able to log in using a different number on the same machine.

**Comment by capt2101akash** [comment]
Same here. Did you use the number for automatic generation of otp?

**Comment by varun-khannaa** [comment]
Facing the same issue, i've been full vaccinated
getting this error since the generation of my certificate
there is another beneficiary who's partially and one who is not at all vaccinated

**Comment by francisbegbie** [comment]
> Same here. Did you use the number for automatic generation of otp?

Yes. I used the Cowin booking extension for Chrome. That was day before yesterday. Didnt log in yesterday. I got this error today. 

Hope the number gets whitelisted.

**Comment by ankursmile** [comment]
Getting the same issue, need to book the COVAXIN 2nd dose but unable to do that.
Please anyone help with the solution.

**Comment by prashantjaiswal** [comment]
I am also not able to login because of the above mentioned issue .. Don't understand why is this been done !

**Comment by mohit-soral** [comment]
Been facing the same issue. Tried with another mobile number, I am able to get the OTP, but for the first mobile number, I keep getting this error.
Details of the beneficiary are linked to the first mobile number alone, so adding as a new beneficiary under a second mobile number would not work !

Is there a directory for the Error codes that could be referred for more information ?

**Comment by pari-25** [comment]
yeah. same issue. "Generate OTP failed"

**Comment by rohanshar** [comment]
They are detecting unusual behavior since 3pm today and blocking phone numbers for 24 hours. Just got off the call with helpline.

**Comment by mohit-soral** [comment]
> They are detecting unusual behavior since 3pm today and blocking phone numbers for 24 hours. Just got off the call with helpline.

@rohanshar Any idea what is the definition of "unusual behavior" here ?

I was not able to find anything in this regards on the official API documentation. Just the usual rate limiting

**Comment by prashantjaiswal** [comment]
@mohit-soral  - I believe anything that pumps up their servers

**Comment by suraj-ar** [comment]
Same thing happened to me. Please let us know when you guys are able to login again

**Comment by hemantsingh011** [comment]
same here

**Comment by hemantsingh011** [comment]
https://www.dqindia.com/cowin-otp-issues-arise-citizens-platform-displays-generate-otp-failed-message/

**Comment by manishdawar** [comment]
Don't know how much time will be taken to fix this bug? Time is running, due date is coming closer and yet to book the 2nd dose of vaccination. Any help how we can get rid of this?

**Comment by rocky-rancho** [comment]
Facing the same issue since yesterday evening, do we know if anybody even monitors these issues on github? How do we amplify it?

**Comment by karan-pahwa** [comment]
Someone on twitter is saying that he talked to the technical team of cowin portal and they are asking to wait for 24 hours then check..
Please check if it is true or not..

**Comment by manishdawar** [comment]
But the issue is when did he talk to the technical team? Did they share any
complaint number or exact resolution time? Time is passing and the due date
is now on head.

*Manish Dawar*

**Comment by prashantjaiswal** [comment]
@cowinapi  - Can you please update when should we expect unblocking on this ..

Beneficiary.validation.wait_to_send_otp
Generate OTP failed

**Comment by prashantjaiswal** [comment]
@cowinapi  - It is giving new error now:

{"errorCode":"USRAUT0030", "error":"Exceeded Search Limit. Please try after some time."}


Did anyone else noticed the same ?

**Comment by suraj-ar** [comment]
> @cowinapi - It is giving new error now:
> 
> {"errorCode":"USRAUT0030", "error":"Exceeded Search Limit. Please try after some time."}
> 
> Did anyone else noticed the same ?

---- YES, I started seeing this error message a few minutes ago

**Comment by rocky-rancho** [comment]
yes, a new error is there - {"errorCode":"USRAUT0030", "error":"Exceeded Search Limit. Please try after some time."}. It's been more than 24 hours now, there is no support system as well, how do we even know the cause of this and avoid this in the future.

**Comment by mohit-soral** [comment]
> @cowinapi - It is giving new error now:
> 
> {"errorCode":"USRAUT0030", "error":"Exceeded Search Limit. Please try after some time."}
> 
> Did anyone else noticed the same ?

Interesting huh ! Yesterday the error code was USRAUT0018 or something, with a very generic message.

**Comment by prashantjaiswal** [comment]
@cowinapi  - This is really disappointing to go to cowin site and finding different issues  while login .. 

Time is running out to get vaccinated and here we are struggling to get an appointment

**Comment by hemantsingh011** [comment]
It's working fine now

**Comment by francisbegbie** [comment]
Yes, I am able to login now. 
It seems they have removed the 'search by district' option.

**Comment by paggy1984** [comment]
> Yes, I am able to login now.
> It seems they have removed the 'search by district' option.

Is it working for you now? by logging into cowin app?
